plugin/titlecase.vim

```diff
@@ -20,6 +22,7 @@ function! s:titlecase(type, ...) abort
       let @i = substitute(@@, WORD_PATTERN, UPCASE_REPLACEMENT, 'g')
       silent execute 'normal! ' . a:type . '`>"ip'
     endif
+    let @@ = regbak
```

Comment:
Thanks for the update, @cspeterson! I think to go just one step further, we'd want to wrap this  in a `try|catch|finally`, specifically doing the restoration bit in the `finally` portion. I'm not sure if `try|finally|endtry` is valid in vimscript (without the `catch`), but that's roughly what I'm thinking.

Could you update to round that out?

Reply:
I believe my recent commit implements what you asked for 🙂